src/layouts/SiteLaunchPad/components/SiteLaunchCheckList.tsx

```diff
@@ -230,20 +249,33 @@ export const SiteLaunchChecklistBody = ({
     </Center>
   ))
 
-  let TABLE_MAPPING: TableMappingProps[] = Array.from(
+  let tableMapping: TableMappingProps[] = Array.from(
     { length: numberOfCheckboxes },
     (_, i) => ({
+      stepNumber: i,
       title: (
-        <Text fontSize="s" {...getTextProps(i + 1, tasksDone)} key={i}>
-          {TITLE_TEXTS[i]}
+        <Text
+          fontSize="s"
+          key={i}
+          textStyle="subhead-2"
+          textColor="base.content.dark"
+          {...getTextProps(i + 1, tasksDone)}
```

Comment:
can i check why we need to pass the `+ 1` here? the `getTextProps` seem to already account for indexing?

Reply:
hey you are right, this can be made clearer a663288b7575ae7d94a4d65d227cab787131dd99